diff --git a/CMakeLists.txt b/CMakeLists.txt
index acb73a5..31c88ac 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -13,6 +13,7 @@ project (uuid_v4)
 
 option(test "Build all tests." OFF)
 option(benchmark "Build benchmarks." OFF)
+option(example "Build example." OFF)
 
 IF (test)
   # build tests (targets: gtest_main, gtest)
@@ -32,5 +33,7 @@ IF (benchmark)
   add_subdirectory (benchmarks)
 ENDIF (benchmark)
 
-add_executable(example example.cpp)
-target_link_libraries (example)
+IF (example)
+  add_executable(example example.cpp)
+  target_link_libraries (example)
+ENDIF(example)
